### ISSUE TITLE
Asthma cured by fitness not always recognized

### ISSUE THREAD

**Issue by keneanung** [svof/svof#158]
reported by Aerek
~~~
SvoF will sometimes not recognize Fitness curing asthma if it takes place in a bunch of queued actions. (I assume it's anti-illusion failsafes, because it doesn't see a prompt within a certain range of lines.)
	- "You inhale deeply, purging yourself of your asthmatic condition."
~~~

**Comment by vadi2** [comment]
This one would be hard to fix without logs - the issue could stay open for
a while.

**Comment by Esync7** [comment]
Can do. This is just the snippet of it actually happening; if context is important, I can provide the full html log, it's just long, ugly, and I was having some runaway script issues. Near as I can tell, this can happen any time you have too many queued actions fire with Fitness in the mix.

`  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell manaleech ast]
Hasar stares at you, giving you the evil eye.
A prickly stinging overcomes your body, fading away into numbness.
Hasar stares at you, giving you the evil eye.
You feel a tightening sensation grow in your lungs.
queue add EQBAL fitness
queue add EQBAL fitness
  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell par manaleech ast]
Hasar eats a ferrum flake.
  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell par manaleech ast]
You must regain balance first.
  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell par manaleech ast]DSL Hasar right leg curare euphorbia
Your eqbal queue is full.
  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell par manaleech ast]
Your eqbal queue is full.
  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell par manaleech ast]
An unspeakable, horrific image rises from the bowels of hell to plague you.
Your mind swims as dizziness overtakes you.
You have recovered balance on all limbs.
[System]: Running queued eqbal command: FITNESS
**You inhale deeply, purging yourself of your asthmatic condition.**
[System]: Running queued eqbal command: FITNESS
You must regain balance first.
[System]: Running queued eqbal command: FITNESS
You burp obscenely.
[System]: Running queued eqbal command: FITNESS
You must regain balance first.
[System]: Running queued eqbal command: FITNESS
"The voices! The voices! Get them out of my head!!" you moan, holding your head in pain.
[System]: Running queued eqbal command: FITNESS
You must regain balance first.
queue add EQBAL fitness
queue add EQBAL fitness
  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell **ast **par manaleech diz]
You flap your arms madly.
(svof): auto-detected stupidity.
queue add EQBAL fitness
queue add EQBAL fitness
  H:4693, M:3899, ecdbk-   0/0|9/6|0/0|11  ( 0, 0, 0, 0, 0, 0,) [hell st par manaleech diz **ast**]`

**Comment by jgh713** [comment]
Again, addressed with #209
